builtin_metadata.json

```diff
@@ -18256,6 +18257,26 @@
     },
     "wasm": false
   },
+  "uuid.parse": {
+    "args": [
+      {
+        "description": "uuid string to be evaluated",
+        "name": "uuid",
+        "type": "string"
+      }
+    ],
+    "available": [
+      "edge"
+    ],
+    "description": "Parses incoming string if valid uuid, returns object with properties if valid",
+    "introduced": "edge",
+    "result": {
+      "description": "another out",
+      "name": "out",
+      "type": "object[any: any]"
+    },
```

Comment:
I may misinterpret how this will end up looking on the web page, but it seems like the "description" and "name" can mo improved a little. Also, the type is `object[string: any]`